packages/ecs/src/UUIDComponent.ts

```diff
@@ -82,6 +82,10 @@ export const UUIDComponent = defineComponent({
       setComponent(entity, UUIDComponent, uuid)
     }
     return state.value
+  },
+
+  generateUUID() {
+    return crypto.randomUUID() as EntityUUID
```

Comment:
Can we revert using this since we've found issues with Node. We can still move generateEntityUUID to here but we should use 'uuid'

Reply:
Yes good point